api/adaptive_load/BUILD

```diff
@@ -0,0 +1,24 @@
+load("@envoy_api//bazel:api_build_system.bzl", "api_cc_py_proto_library")
+
+licenses(["notice"])  # Apache 2
+
+api_cc_py_proto_library(
+    name = "base",
```

Comment:
Can we use a more descriptive name here? E.g. adaptive_load_proto or similar.

Reply:
Done.